`cargo build --test` fails on Windows

[One of the tests](https://github.com/bitflags/bitflags/blob/8a10bdc144bb2a4b97c63cfa76be90228954d029/src/lib.rs#L920) for `bitflags-1.1.0` fails to compile on Windows with a stable rustc (version 1.36.0).

```
[INFO] [stderr] error[E0425]: cannot find value `_CFG_A` in this scope
[INFO] [stderr]    --> src\lib.rs:927:28
[INFO] [stderr]     |
[INFO] [stderr] 927 |             const _CFG_C = _CFG_A.bits | 0b10;
[INFO] [stderr]     |                            ^^^^^^ not found in this scope
[INFO] [stderr] 
[INFO] [stderr] error: aborting due to previous error
```

This suggests that `#[cfg]` attributes are not being properly added to `const` declarations, but I'm not quite sure exactly what's going on here.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -919,12 +919,12 @@ mod tests {
 
     bitflags! {
         struct _CfgFlags: u32 {
-            #[cfg(windows)]
-            const _CFG_A = 0b01;
             #[cfg(unix)]
-            const _CFG_B = 0b01;
+            const _CFG_A = 0b01;
             #[cfg(windows)]
-            const _CFG_C = _CFG_A.bits | 0b10;
+            const _CFG_B = 0b01;
+            #[cfg(unix)]
+            const _CFG_C = Self::_CFG_A.bits | 0b10;
         }
     }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1111,7 +1111,7 @@ mod tests {
     #[cfg(bitflags_const_fn)]
     #[test]
     fn test_const_fn() {
-        const M1: Flags = Flags::empty();
+        const _M1: Flags = Flags::empty();
 
         const M2: Flags = Flags::A;
         assert_eq!(M2, Flags::A);

